### ISSUE TITLE
Support for user commands

### ISSUE THREAD

**Issue by kaypoush** [airdcpp-web/airdcpp-webclient#430]
**Is your feature request related to a problem? Please describe.**
I cannot use user commands like requesting a release in a hub. It's possible with the Windows version.

**Describe the solution you'd like**
When you right click on a hub, it should give me the possibility to interact with that hub trough user commands.

**Describe alternatives you've considered**
There is no alternative except asking a friend who uses a Windows version to request a release for you.

**Additional context**
I'm using AirDC++w 2.11.3b-26-g9a31 on Manjaro Linux.

**Comment by maksis** [comment]
> There is no alternative except asking a friend who uses a Windows version to request a release for you.

That's not really true. Those are just regular text commands that can be sent also by using context menus. You should check the list of commands that are available in the hub (or ask the operators for assistance if you can't find those).

**Comment by kaypoush** [comment]
> > There is no alternative except asking a friend who uses a Windows version to request a release for you.
> 
> That's not really true. Those are just regular text commands that can be sent also by using context menus. You should check the list of commands that are available in the hub (or ask the operators for assistance if you can't find those).

Indeed, thanks for the tip. Typing ```!help``` in the main channel gives me a list of available commands for the hub. For example, ```!request add <release>``` allows me to do a request. The difference with the Windows version is that they have a contextual menu that will have a quick link to all the commands just like you said. In that case, this is probably a duplicate of https://github.com/airdcpp-web/airdcpp-webclient/issues/116 and should be closed.

**Comment by maksis** [comment]
> The difference with the Windows version is that they have a contextual menu that will have a quick way to use the commands just like you said. In that case, this is probably a duplicate of https://github.com/airdcpp-web/airdcpp-webclient/issues/116 and should be closed.

User commands and context menus are separate features
